mergify_engine/queue/merge_train.py

```diff
@@ -355,7 +361,9 @@ def deserialize(
             last_checks=last_checks,
             last_evaluated_conditions=data.get("last_evaluated_conditions"),
             has_timed_out=data.get("has_timed_out", False),
+            checks_timeout=data.get("checks_timeout"),
```

Comment:
Since `checks_timeout` is computed from the configuration we should avoid storing it in Redis. In case of the user value change, this value will not be updated. You can get the config with something like:

```
rules = self.train.get_queue_rules()
rule = rules[self.initial_embarked_pulls[0].config["name"]]
```